src/backend/distributed/commands/table.c

```diff
@@ -701,13 +701,81 @@ PostprocessAlterTableSchemaStmt(Node *node, const char *queryString)
 }
 
 
+static char *
+GenerateIndexConstraintName(const char *tabname, Oid namespaceId, Constraint *constraint)
```

Comment:
Maybe simply GenerateConstraintName here
Also we need a comment for this method: there you can explain the types of constraint names that we currently support within this function